communications/meetings.md

```diff
@@ -0,0 +1,67 @@
+# Meetings
+
+Use the guides below as a starting point, and adapt for specific situations accordingly.
+
+## Before A Meeting
+
+Objectives:
+
+- Determine the scope and minimum length of the meeting.
+- Give all participants as much context as possible to make an informed decision on RSVP.
+- Gather topics and issues for agenda items that are relevant to the scope of the meeting,
+done either individually or through crowdsourcing to increase accuracy and minimize bias
+- Asynchronously resolve as many agenda items as possible before the meeting.
+
+Guide:
+
+- Send a calendar invite to all participants, with Zoom/Hangouts Meet.
+- Create the meeting agenda on Hugo for the meeting, provide sufficient time ahead of the meeting to accommodate all participants.
+    - For a net-new meeting, the organizer(s) should define the initial scope of agenda items.
+    - For a recurring meeting with past sessions, use the agenda/notes items from the previous meeting session as a starting point. Remove ones no longer needed or already resolved.
+- Initiate one or more meeting note(s) on Hugo for meeting agenda discussions.
+    - Send these notes to the relevant Slack channel.
+    - Edit meeting agenda to reflect the outcome of these discussions.
+- In each meeting note, participants write down briefing, information, knowledge, and ideas that are relevant.
+    - For a recurring meeting with past sessions, focus on briefings of pending items from the previous meeting session, such as what each _has done_, _has been doing_, and _plans to do_. Give everyone else a sense of _where you are_ and _the direction you're heading to_. This helps others to determine if there are potential directional conflicts/blocks.
+    - Raise relevant questions, problems, obstacles, unknowns, and any other issues. Try to make them demonstratable, by leveraging modern tools such as screen-captured images or videos, and/or steps to reproduce these issues.
+    - Preemptively discuss. Seek to resolve as many issues and directional conflicts/blocks as possible.
+- Cross off resolved (or portions of) items from the list of the meeting agenda. In the event of having all the item discussions resolved, skip to [section: After A Meeting](#after-a-meeting).
+- Put a tentative timebox on each unresolved item. This can be an outcome of a collective agreement.
```

Comment:
Would recommend having a line about after agenda is set, initiate a new note on Hugo for meeting notes to be used during the meeting.

Suggestion:
'Once agenda is finalized, initiate a note on Hugo for meeting notes titled Meeting Notes and share with the relevant Slack channel'

Reply:
my intention is to actually keep using the per-item notes to be used during meeting. i may need to further clarify/reword to emphasize that.

Reply:
I think if you mix in meeting notes with the agenda discussion, it can get very confusing. May be better to create one note for meeting documentation. 

One possible workflow is that during the meeting, one person will update the meeting note. However, the rest of the participants can comment on 2 possible places:
a) the note dedicated to the agenda item
b) the meeting note
Better to choose one and not both. Then whoever is updating the meeting note, can transfer the comments from the agenda item into the meeting note. Does this sound feasible?